tests_e2e/tests/extensions_disabled.py

```diff
@@ -81,6 +82,11 @@ def run(self):
             .is_greater_than(pytz.utc.localize(disabled_timestamp))
         log.info("The VM Agent reported status after extensions were disabled, as expected.")
 
+    def get_ignore_error_rules(self) -> List[Dict[str, Any]]:
+        return [
+            {'message': 'No handler status found for Microsoft.Azure.Extensions.CustomScript'},
```

Comment:
This message showed up in today's daily run. Will be there or not depending on the time of the test